main_full_par_trafo.g

```diff
@@ -52,6 +52,8 @@ ChiefParallel := function( galoisField,mat,a,b )
 			UpdateRowTrafoInput;
 
     ##Preparation: Init and chopping the matrix mat
+    Info(InfoGauss, 1, "------------ Start ChiefParallel in main_par_trafo.g ------------");
```

Comment:
We don't need the filename. Also check whether the line then is longer than 80 characters.